src/SpendShelf.BankTransactionsImport.Tests/TransactionsParser/BankTransactionsDateParserTests.cs

```diff
@@ -0,0 +1,34 @@
+﻿using SpendShelf.BankTransactionsImport.TransactionsParser;
+using System;
+using System.Collections.Generic;
+using Xunit;
+
+namespace SpendShelf.BankTransactionsImport.Tests.TransactionsParser
+{
+    public class BankTransactionsDateParserTests
+    {
+        [Theory]
+        [MemberData(nameof(Data))]
+        public void Parse_When_Valid(string value1, string value2, DateTime expected)
```

Comment:
I don't quite understand what naming convention you use here
Pls use some from this list:
https://dzone.com/articles/7-popular-unit-test-naming

I personally recommend 1st or 2nd

Reply:
Fixed, please check.